docs/community/release-checklist.md

```diff
@@ -47,54 +47,103 @@ In general, changes to anything a user might reasonably link to, customize, or i
 be backward-compatible, or else require a major release. aks-engine users can be confident that upgrading
 to a patch or to a minor release will not break anything.
 
-## How to Release aks-engine
+## Creating a New Release
 
-This section leads a maintainer through creating an aks-engine release.
+Let's go through the process of creating a new release of [aks-engine][].
 
-### Step 1: Assemble Master Changelog
+We will use **v0.32.3** as an example herein. You should replace this with the new version you're releasing.
```

Comment:
would it be easier to add a bash var `VERSION=v0.32.3` and then use that in the rest of the commands so they can be easily copied/pasted?

Reply:
Good idea—done.